src/main/pg/rates.c

```diff
@@ -67,6 +67,14 @@ void pgResetFn_controlRateProfiles(controlRateConfig_t *controlRateConfig)
             .accel_limit[FD_PITCH] = 0,
             .accel_limit[FD_YAW] = 0,
             .accel_limit[FD_COLL] = 0,
+            .setpoint_boost_gain[FD_ROLL] = 35,
```

Comment:
I am not sure we want to enable this feature by default.
Lost of users are already considering RF too snappy.

Reply:
All changed to 0